login-workflow/example/src/screens/ExampleHome.tsx

```diff
@@ -22,20 +22,31 @@ import ExitToApp from '@mui/icons-material/ExitToApp';
 import Avatar from '@mui/material/Avatar';
 import AppBar from '@mui/material/AppBar';
 import Toolbar from '@mui/material/Toolbar';
+import LockIcon from '@mui/icons-material/Lock';
 import Typography from '@mui/material/Typography';
 import { useTheme } from '@mui/material/styles';
 import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
 import * as Colors from '@brightlayer-ui/colors';
 import FormControl from '@mui/material/FormControl';
 import { MenuItem, Select, SelectChangeEvent } from '@mui/material';
+import { ChangePasswordDialog } from '@brightlayer-ui/react-auth-workflow';
 
 export const ExampleHome: React.FC<React.PropsWithChildren> = () => {
     const app = useApp();
     const { t, i18n } = useTranslation();
     const [open, setOpen] = useState(false);
     const navigate = useNavigate();
     const theme = useTheme();
+    const [openDialog, setOpenDialog] = React.useState(false);
+
+    const dialogClose = (): void => {
```

Comment:
No need to write this function, as it is getting used only once. You can directly setOpenDialog(false)